### ISSUE TITLE
Typo in examples/causal_language_modeling/peft_lora_clm_accelerate_ds_zero3_offload.py

### ISSUE THREAD

**Issue by rmill040** [huggingface/peft#275]
There is a small typo in the deepspeed stage 3 example for CLM. In this [line](https://github.com/huggingface/peft/blob/445940fb7b5d38390ffb6707e2a989e89fff03b5/examples/causal_language_modeling/peft_lora_clm_accelerate_ds_zero3_offload.py#L270), the code is supposed to calculate the overall `train_epoch_loss` but uses the wrong dataloader as the denominator. In other words, it is currently:
```python
train_epoch_loss = total_loss / len(eval_dataloader)
```
but should be 
```python
train_epoch_loss = total_loss / len(train_dataloader)
```

**Comment by pacman100** [comment]
Hello, nice catch, do you want to raise a PR fixing the same?

**Comment by rmill040** [comment]
@pacman100 Sure thing!

PR is in #277

**Comment by younesbelkada** [comment]
Closing as fixed in #277
